tests/acceptance/features/lib/FirstrunwizardPage.php

```diff
@@ -0,0 +1,54 @@
+<?php
+/**
+ * ownCloud
+ *
+ * @author Bhawana Prasain  <bhawana.prs@gmail.com>
+ * @copyright Copyright (c)  Bhawana Prasain 2019 bhawana.prs@gmail.com
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU AFFERO GENERAL PUBLIC LICENSE
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU AFFERO GENERAL PUBLIC LICENSE for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public
+ * License along with this library.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+namespace Page;
+
+use \Behat\Mink\Element\NodeElement;
+
+/**
+ * Page object for Firstrunwizard
+ *
+ */
+
+class FirstrunwizardPage extends OwncloudPage {
+	protected $fileXpath = "//*[@id='closeWizard']";
+
+	protected $secfileXpath = "//*[@id='firstrunwizard']";
```

Comment:
Can you please rename this to a better variable name?

My suggestion would be:
```suggestion
	protected $firstRunWizardPopupXpath = "//*[@id='firstrunwizard']";
```